### ISSUE TITLE
Licensing

### ISSUE THREAD

**Issue by trickeydan** [srobo/nemesis#9]
Does this repo have / need a Open Source software licence? Do we have explicit permission to use it currently?

**Comment by PeterJCLaw** [comment]
Yes, we have permission to use it currently. (I have emails from the contributors regarding their preferences for a licence, but haven't got around to applying it to the codebase yet; it's a bit complicated given the submodule situation at the moment)
